source/backend/database/models/usersModel.js

```diff
@@ -12,4 +12,34 @@ async function getByUserId(userId) {
   return result;
 }
 
-module.exports = { getByUserId };
+/**
+ * Searches all completed recipes by userId
+ * @param userId
+ * @returns {Promise<awaited Knex.QueryBuilder<TRecord, TResult>>}
+ */
+async function getCompletedRecipes(userId){
+  const completedRecipes = await db('completedRecipes')
+    .innerJoin('recipes', function() {
```

Comment:
use syntax: `innerJoin('recipes', 'completeRecipes.userId', 'recipes.recipeId')` as it will be more readable then using callback functions

Reply:
fixed.